### PR TITLE
big refactor to remove bad past decisions

### PR DESCRIPTION
**Pull request by george-silva**

### CODE REVIEW

File: src/planscape/planning/tests/helpers.py [hunk 1]
```diff
@@ -17,10 +17,12 @@ def _create_planning_area(
     Creates a planning_area with the given user, name, geometry, notes.  All regions
     are in Sierra Nevada.
     """
+    from planning.models import RegionChoices
+
     planning_area = PlanningArea.objects.create(
         user=user,
         name=name,
-        region_name="sierra-nevada",
+        region_name=RegionChoices.SIERRA_NEVADA,
```
Comment (pflopez) [inline src/planscape/planning/tests/helpers.py:25]: 👏

File: src/planscape/planning/views.py [hunk 2]
```diff
@@ -62,32 +62,6 @@ def _convert_polygon_to_multipolygon(geometry: dict):
     return actual_geometry
 
 
-# TODO: Along with PlanningAreaSerializer, refactor this a bit more to
-# make it more maintainable.
-def _serialize_planning_area(
```
Comment (pflopez) [inline src/planscape/planning/views.py:67]: byeee

**Review by pflopez — APPROVED** [review]

File: src/planscape/planning/serializers.py [hunk 3]
```diff
@@ -1,30 +1,31 @@
...
 from rest_framework_gis import serializers as gis_serializers
 from django.conf import settings
-from collaboration.models import Permissions
 from collaboration.services import get_role, get_permissions
 from planning.models import PlanningArea, Scenario, ScenarioResult, SharedLink
 from planning.services import get_acreage
 from stands.models import StandSizeChoices
 
 
 # TODO: flesh all serializers more for better maintainability.
```
Comment (lastminutediorama) [inline src/planscape/planning/serializers.py:10]: Maybe we don't need this TODO now?

**Review by lastminutediorama — APPROVED** [review]
looks great, thanks for cleaning this up. this will make life easier.

**Pull request merged**